crates/extensions/c8y_mapper_ext/src/actor.rs

```diff
@@ -61,7 +61,7 @@ impl Actor for C8yMapperActor {
     async fn run(&mut self) -> Result<(), RuntimeError> {
         let init_messages = self.converter.init_messages();
         for init_message in init_messages.into_iter() {
-            let _ = self.mqtt_publisher.send(init_message).await?;
+            self.mqtt_publisher.send(init_message).await?;
```

Comment:
It's a really bad practice actually. If you include `let _ = `, it will stop rust from generating any warnings about the return value (as you're explicitly allowing it to be anything), so if the `?` is removed, you'll never get warned about the non-handling of the error. You should only be using `let _ =` when you want to explicitly ignore an error.

Reply:
Unfortunately, we have many of those.

```
thin-edge.io git:(main) ✗ rg 'let _ =' | wc -l           
113
```